src/main/java/com/iemr/common/controller/users/IEMRAdminController.java

```diff
@@ -429,8 +440,9 @@ public String setPassword(
 			}
 			User mUser = mUsers.get(0);
 			String setStatus;
-			noOfRowModified = iemrAdminUserServiceImpl.setForgetPassword(mUser, m_user.getPassword(),
-					m_user.getTransactionId(),m_user.getIsAdmin());
+			String decryptPassword = aesUtil.decrypt("Piramal12Piramal", m_user.getPassword());
```

Comment:
@ravishanigarapu The username seems to be hardcoded here. Please check and correct.